xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Translations.xml

```diff
@@ -44,6 +44,7 @@ help.tipsPanel.improveXWikiTip=Did you know that you can improve XWiki? Take 5 m
 help.tipsPanel.breadcrumbsTip=You can click on the arrows next to the breadcrumb elements to quickly navigate to sibling and children pages.
 help.tipsPanel.advancedHiddenShortcutTip=Press the x+x+x+a key combo to toggle the [[Advanced mode&gt;&gt;https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/PageEditing#HSimpleandAdvancededitingmodes]] and x+x+x+h to toggle the display of [[hidden pages&gt;&gt;https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/PageEditing#HHidingPages]].
 help.tipsPanel.linkAutocompleteWYSIWYGTip=When editing in WYSIWYG, type ##[## and then start typing letters to get suggestions for creating a link.
+help.tipsPanel.WYSIWYGEditorTip=When editing in WYSIWYG mode, the ##Alt + 0## shortcut opens a help modal with information about using the keyboard to navigate the editor.
```

Comment:
While all other tips do the same, it is actually [not a best practice to have markup in translations](https://dev.xwiki.org/xwiki/bin/view/Community/L10N/Conventions/#HRulesfortranslatingcompositeandlongmessages). Instead, the markup should be provided as a parameter. I think it would be good to not introduce further translations that don't follow our rules.

Independent of that, it should also be mentioned that this is `⌥0` on Mac (it is documented like that in CKEditor 5, according to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/altKey), `altKey` is "Option" or "⌥" on Mac, so this should also work on CKEditor 4).

Reply:
> While all other tips do the same, it is actually [not a best practice to have markup in translations](https://dev.xwiki.org/xwiki/bin/view/Community/L10N/Conventions/#HRulesfortranslatingcompositeandlongmessages). Instead, the markup should be provided as a parameter. I think it would be good to not introduce further translations that don't follow our rules.

:+1: Fixed in https://github.com/xwiki/xwiki-platform/pull/3149/commits/5dd71d06248c9a3d5ce0aa5d5051a867533f991c

> Independent of that, it should also be mentioned that this is ⌥0 on Mac (it is documented like that in CKEditor 5, according to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/altKey), altKey is "Option" or "⌥" on Mac, so this should also work on CKEditor 4).

I updated our [Keyboard shortcut documentation](https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/KeyboardShortcuts) with this info.

Fixed in https://github.com/xwiki/xwiki-platform/pull/3149/commits/137938d4c4a88b0f5234e5a7a24f1917494fd92c :+1: 

Here is what the Panel looks like after those changes 
![image](https://github.com/xwiki/xwiki-platform/assets/28761965/49013362-19dc-4f3e-b26b-bb9b6ed4e00c)

Reply:
Option is another name for "⌥" on Mac, this feels a bit verbose. To make things even more confusing, apparently outside the US the key is also labelled "Alt" according to [this article](https://www.macworld.com/article/670647/what-is-the-option-key-on-a-mac.html). I don't have a Mac, so I can't confirm. I would suggest to just include "⌥0" [similar to the CKEditor 5 documentation](https://ckeditor.com/docs/ckeditor5/latest/features/accessibility.html#user-interface-and-content-navigation-keystrokes). Also, it is not a good practice to use the same parameter for both start and end. Imagine you wanted to use this message in an HTML context - how would you inject the start and end tags?

Reply:
> Also, it is not a good practice to use the same parameter for both start and end. Imagine you wanted to use this message in an HTML context - how would you inject the start and end tags?

I added a sentence about this on https://dev.xwiki.org/xwiki/bin/view/Community/L10N/Conventions/#HRulesfortranslatingcompositeandlongmessages :+1: 

__
Both comments addressed in https://github.com/xwiki/xwiki-platform/pull/3149/commits/e99496c475de6cbee493ddc486d38efdcfd9fc90 :+1: